client/src/components/RadioInput.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import PropTypes from 'prop-types';
+
+const Container = styled.div`
```

Comment:
This looks like a generic component that might be useful in other places of the app in the future. Maybe it makes sense to move this into a new file.

Reply:
This is true and I will use it as a separate component in another place 🤗